src/main/java/mate/academy/UserService.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy;
+
+import mate.academy.model.User;
+
+public class UserService {
+
+    public User findByEmail(String email) {
+        if ("bob@i.ua".equals(email)) {
+            return new User("bob@i.ua", "1234");
+        } else if ("alice@i.ua".equals(email)) {
+            return new User("alice@i.ua", "1234");
+        } else {
+            return null;
+        }
```

Comment:
very bad realization with tons of code duplicating. use data structure for storing users and iterate through users to find wanted one